literatur.bib

```diff
@@ -9270,6 +9270,14 @@ @InProceedings{jinnai-fukunaga-aaai2017
   crossref =     "aaai2017"
 }
 
+@InProceedings{joergensen-et-al-icaart2025,
+  author =       "Oliver Joergensen and Vy Le and Tin Nguyen and Ginel Dorleon",
+  title =        "Optimizing Elevator Performance with {SARL} Multi-Agent Systems: A Distributed Approach for Enhanced Responsiveness and Efficiency",
+  booktitle =    "Proceedings of the International Conference on Agents and Artificial Intelligence (ICAART)",
```

Comment:
Can you please add a long entry for the conference in crossref.bib and an abbreviated one in crossref-short.bib? See the other examples as a reference. Also, the booktitle should state "Proceedings of the Xth International ..." with X substituted for the correct number, of course.